modules/mobile-tests/helpers/constants.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable no-undef */
+
+module.exports = {
+    APPIUM_VERSION: process.env.APPIUM_VERSION || '1.22.3',
+    // eslint-disable-next-line max-len
+    APP_PATH_SIMULATOR: 'Users/cchitu/Library/Developer/Xcode/DerivedData/jitsi-meet-gpsplsfarleyxebhmcufemivqqbl/Build/Products/Debug-iphonesimulator/jitsi-meet.app',
+    AUTOMATION_NAME: process.env.AUTOMATION_NAME,
+
+    ANDROID_DEVICE: driver.isAndroid,
```

Comment:
There is a problem with the driver object because I keep getting 'driver is not defined' even though it's part of webdriverio. I cannot access it and I can't find a solution to use it.